### PR TITLE
doc for quick start, exec/logs/port-forward; add render-only to init cmd

### PR DESCRIPTION
**Pull request by hongchaodeng**
ref https://github.com/oam-dev/kubevela/issues/441

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -139,6 +139,7 @@ core-uninstall: manifests
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=charts/vela-core/crds
+	./hack/vela-templates/gen_definitions.sh
```
Comment (wonderflow) [inline Makefile:142]: 👍 I was just going to add it in my PR,haha

File: pkg/appfile/appfile.go [hunk 2]
```diff
@@ -21,6 +21,7 @@ var (
 )
 
 const DefaultAppfilePath = "./vela.yaml"
+const DefaultVersion = "1.0-alpha.1"
```
Comment (wonderflow) [inline pkg/appfile/appfile.go:24]: for a version of format, we usually name it like  `v1alpha1` (k8s) or `2010-09-09`(cloud formation)

File: pkg/commands/init.go [hunk 3]
```diff
@@ -72,6 +75,20 @@ func NewInitCommand(c types.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
...
+			if err != nil {
+				return err
+			}
+			err = ioutil.WriteFile("./vela.yaml", b, 0600)
+			if err != nil {
+				return err
+			}
+			o.IOStreams.Info("\nRendering deployment config to " + color.New(color.FgCyan).Sprint("vela.yaml"))
+
+			if o.renderOnly {
```
Comment (wonderflow) [inline pkg/commands/init.go:88]: the renderOnly check should move before `oam.BaseRun`(line 73), or it's too late to check.

File: pkg/appfile/appfile.go (hunk 2 above)
Comment (hongchaodeng) [reply pkg/appfile/appfile.go:24]: v1alpha1 is k8s specific and it is not even semver.
See https://docs.docker.com/compose/:

```
version: "3.8"
services:
```

Comment (wonderflow) [reply pkg/appfile/appfile.go:24]: yes, something like `3.8` is also good for me.

Comment (hongchaodeng) [reply pkg/appfile/appfile.go:24]: Using `v1alpha1` will complicate developers who are not familiar with k8s

Comment (wonderflow) [reply pkg/appfile/appfile.go:24]: what about just `v1`?

File: pkg/commands/init.go (hunk 3 above)
Comment (hongchaodeng) [reply pkg/commands/init.go:88]: fixed

File: pkg/appfile/appfile.go (hunk 2 above)
Comment (hongchaodeng) [reply pkg/appfile/appfile.go:24]: It is unwise to declare v1 for now...
This is not something we want to solve before kubecon. I would suggest let's just follow compose spec for now.

Comment (hongchaodeng) [reply pkg/appfile/appfile.go:24]: Since it is alpha we can change that later

Comment (hongchaodeng) [reply pkg/appfile/appfile.go]: I removed the `version` field from Appfile. 
This is not something deserved a place right now

File: docs/quick-start.md [hunk 4]
```diff
@@ -1,9 +1,129 @@
...
+
+Applying deploy configs ...
+Checking if app has been deployed...
+app existed, updating existing deployment...
+✅ app has been deployed 🚀🚀🚀
+    Port forward: vela port-forward testapp
+             SSH: vela exec testapp
+         Logging: vela logs testapp
+      App status: vela app status testapp
+  Service status: vela svc status testsvc
```
Comment (wonderflow) [inline docs/quick-start.md:75]: Here was changed, only `vela status` is enough

**Review by wonderflow — APPROVED** [review]

**Pull request merged**